### PR TITLE
Creating and adding tags to a new product

### PR DESCRIPTION
**Pull request by edelveiss**
# Description
User can create a new tag and add it to a new product.
User can add an existing tag to a new product.

## Type of change
- [x] New feature (non-breaking change which adds functionality)
## Change Status
- [x] Complete, tested, ready to review and merge

# Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] There are no merge conflicts

### CODE REVIEW

**Comment by edelveiss** [comment]
[Creates and adds tags to a new product](https://trello.com/c/ssqv0cdI/55-creates-and-adds-tags-to-a-new-product)

**Review by aaronmatson1 — APPROVED** [review]
Looks good! keep it up!

File: src/components/sellerPages/inventory/index.js [hunk 1]
```diff
@@ -105,6 +120,7 @@ function Inventory({
           onClick={() => {
             console.log(mainInfo);
```
Comment (Dustnclay) [inline src/components/sellerPages/inventory/index.js:121]: you can remove console logs if you are done,

**Review by Dustnclay — APPROVED** [review]
as always, looks great!

**Comment by Dustnclay** [comment]
everything is great, incredible job

**Pull request merged**